lib/modules/userTagger.js

```diff
@@ -595,7 +595,11 @@ addModule('userTagger', {
 			}
 			document.getElementById('userTaggerColor').selectedIndex = 0;
 		}
-		this.userTaggerToolTip.setAttribute('style', 'display: block; top: ' + thisXY.y + 'px; left: ' + thisXY.x + 'px; z-index: 20;');
+		if (thisXY.x + 334 < window.innerWidth) {
```

Comment:
I originally did this, but offsetWidth returns 0 because it's not visible when called, it returns properly using jquery, but then it doesn't pass the built in test. I'll revise this however and find a suitable solution.